Cargo.toml

```diff
@@ -9,12 +9,13 @@ license = "MIT"
 readme = "README.md"
 
 [dependencies]
+const-oid = { version = "0.9.6", features = ["db"] }
```

Comment:
Can you add `default-features = false` here as well? This protects against `const-oid` adding default features in a minor (or even patch) release, which isn't considered semver-breaking if nothing was removed or changed in the API, but would pull in additional dependencies that we might not need.

Reply:
Done in the PR I opened.